src/NuGet.Services.AzureSearch/VersionList/VersionListData.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NuGet.Services.AzureSearch
+{
+    public class VersionListData
+    {
+        [JsonConstructor]
+        public VersionListData(Dictionary<string, VersionPropertiesData> versionProperties)
+        {
+            VersionProperties = new Dictionary<string, VersionPropertiesData>(
+                versionProperties,
+                StringComparer.OrdinalIgnoreCase);
+        }
+
+        /// <summary>
+        /// A dictionary of all versions currently available for this package. This includes listed and unlisted
+        /// versions. The key is the full version string (can include build metadata). The look-up is case-insensitive,
```

Comment:
>The look-up is case-insensitive [](start = 87, length = 31)

Should we have a unit test on this?

Reply:
Oh, I meant to remove this behavior and only use `VersionListData` for serialization concerns.

I'll remove this.